travis/Dockerfile-travis-watch-build

```diff
@@ -0,0 +1,29 @@
+FROM node:6.2
+MAINTAINER ODL DevOps <mitx-devops@mit.edu>
+
+# this dockerfile builds the hub image for the watch container
+# we don't use this directly, instead we push to docker-hub, and
+# then Dockerfile-travis-watch uses that pushed image to bootstrap
+# itself
+
+RUN apt-get update && apt-get install libelf1
+
+RUN mkdir /src
+
+WORKDIR /src
+
+RUN adduser --disabled-password --gecos "" mitodl
+
+RUN chown -R mitodl:mitodl /src
+
+USER mitodl
+
+COPY package.json /src
```

Comment:
Just made that change, then built and pushed a new image for `mitodl/mm_watch_travis`. The next build on this branch should pull that image.